src/sentry/api/endpoints/organization_missing_org_members.py

```diff
@@ -125,7 +122,7 @@ def provider_reducer(dict, integration):
             if integration_provider != "github":
                 continue
 
-            queryset = self._get_missing_members(
+            queryset = _get_missing_organization_members(
```

Comment:
In most cases this would be empty so you can early exit here

Reply:
i do have to return something even if it's empty though

Reply:
i can do extra processing only if the queryset exists